app/file.go

```diff
@@ -593,6 +593,13 @@ func (a *App) UploadFileX(channelId, name string, input io.Reader,
 		return t.fileinfo, aerr
 	}
 
+	// attempt to extract width & height out of svg
```

Comment:
This should be moved to preprocessImage. There is a similar exception made there for GIFs.